karma.conf.js

```diff
@@ -101,7 +101,7 @@ module.exports = function(config) {
 
 		files: [
 			{ pattern: 'test/polyfills.js', watched: false },
-			{ pattern: config.grep || '{debug,hooks,compat,test-utils,}/test/{browser,shared}/**.test.js', watched: false }
+			{ pattern: config.grep || '{debug,hooks,compat,test-utils,}/test/{browser,shared}/suspense.test.js', watched: false }
```

Comment:
If you only want to run a specific group of tests you can always use `.only` like `it.only()` or `describe.only()`. It may not speed up the globbing process as much as this change here :)

Reply:
Good point. Totally forgot about only :)

Reply:
I believe there's also `npm t -- --grep suspense`.